form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/web/rest/dto/FormFlowProcessLinkCreateRequestDto.kt

```diff
@@ -20,9 +20,11 @@ import com.fasterxml.jackson.annotation.JsonTypeName
 import com.ritense.processlink.domain.ActivityTypeWithEventName
 import com.ritense.processlink.web.rest.dto.ProcessLinkCreateRequestDto
 import com.ritense.valtimo.formflow.mapper.FormFlowProcessLinkMapper.Companion.PROCESS_LINK_TYPE_FORM_FLOW
+import java.util.UUID
 
 @JsonTypeName(PROCESS_LINK_TYPE_FORM_FLOW)
 data class FormFlowProcessLinkCreateRequestDto(
+    override val processLinkId: UUID = UUID.randomUUID(),
```

Comment:
This is a breaking change. For kotlin moving this to the end of the properties would avoid breaking, but for Java this will always be a breaking change. Maybe we should add this in major 11?